src/adapter/src/config/frontend.rs

```diff
@@ -49,12 +53,17 @@ impl SystemParameterFrontend {
         let ld_config = ld::ConfigBuilder::new(ld_sdk_key)
             .event_processor(ld::EventProcessorBuilder::new().on_success({
                 let last_known_time_seconds = ld_metrics.last_known_time_seconds.clone();
+                let last_cse_time_seconds = ld_metrics.last_cse_time_seconds.clone();
                 Arc::new(move |result| {
                     if let Ok(t_next) = u64::try_from(result.time_from_server / 1000) {
                         let t_curr = last_known_time_seconds.get();
```

Comment:
I will remove the old metric once we deploy this change and update the relevant Prometheus clients refer to `last_cse_time_seconds` instead of `last_known_time_seconds`.